lib/rules/object-shorthand.js

```diff
@@ -100,6 +103,7 @@ module.exports = {
         const PARAMS = context.options[1] || {};
         const IGNORE_CONSTRUCTORS = PARAMS.ignoreConstructors;
         const AVOID_QUOTES = PARAMS.avoidQuotes;
+        const ALLOW_EXPLICIT_RETURN_ARROWS = typeof PARAMS.allowExplicitReturnArrows === "undefined" || PARAMS.allowExplicitReturnArrows;
```

Comment:
In my opinion, an inverted option name is more confusing because it causes a double negative. For example, I have to think for a second to figure out what `disallowExplicitReturnArrows: false` means, but this isn't the case for `allowExplicitReturnArrows: true`.

That said, I don't really feel strongly about this either way.